web/packages/teleterm/src/preload.ts

```diff
@@ -164,3 +168,21 @@ async function withErrorLogging<ReturnValue>(
     throw e;
   }
 }
+
+/**
+ * Returns a copy of `TshdClient` with insecure methods disabled
+ * to prevent access from the untrusted renderer process.
+ *
+ * This is possible thanks to the context bridge and the `cloneClient` function,
+ * which selectively clones only RPC methods and nothing more.
+ * As a result, disabled methods are inaccessible via the `tshdClient` prototype.
```

Comment:
I'd say this:

> As a result, disabled methods are inaccessible in the renderer process since the prototype of `tshdClient` is not shared with the renderer process. The renderer process also does not receive the ability to start a new arbitrary client, which could then be used to circumvent this protection.